apps/snapodds/src/app/odds/odds.component.html

```diff
@@ -51,11 +51,12 @@
       ></snapodds-odds-line>
     </div>
 
-    <div
+    <a
       class="c-game"
-      [class.c-game--clickable]="sportsBook.redirectUrl"
       *ngFor="let sportsBook of lineOdds.sportsBooks"
-      (click)="openSportsBookRedirectUrl($event, sportsBook.redirectUrl)"
```

Comment:
May be we need to have a click handler to prevent default action if no redirect URL available which is required for best odds.